build/develop-env/develop-env/Dockerfile

```diff
@@ -127,12 +131,44 @@ RUN source /home/saluser/.setup_salobj.sh && \
     setup ts_standardscripts -t current && \
     scons version || echo "Cannot generate scons version."
 
+WORKDIR /home/saluser/repos/ts_auxtel_standardscripts
+RUN source /home/saluser/.setup_salobj.sh && \
+    setup lsst_distrib && \
+    export LSST_DDS_IP=`ip route get 1 | awk '{print $7;exit}'` && \
```

Comment:
Is there a reason in particular that this line is needed here? None of the other packages have it and I believe salobj and SAL should be setup in .setup_salobj.sh.

Reply:
good catch, I might have been experimenting with the Dockerfile and left that in? I don't remember. I'll take it out.